editor/src/components/context-menu-wrapper.tsx

```diff
@@ -2,7 +2,16 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/react'
 import fastDeepEquals from 'fast-deep-equal'
-import React, { Component as ReactComponent } from 'react'
+import React, {
+  type CSSProperties,
```

Comment:
minor - we need to decide at some point whether we're going with `React.<Something>` or `import { Something } from 'react'`

Reply:
Roger that, feels weird to me, but I gotta learn all the preferences here, so grill me on them all! Thanks.